### ISSUE TITLE
If max_attacks>1, automatically re-select the unit

### ISSUE THREAD

**Issue by Kwandulin** [wesnoth/wesnoth#2668]
If a unit has max_attacks > 1 and attacks, it would be great, if the unit is automatically re-selected and the red arrows to attack nearby units show up again. I've attached an image to show the screen I am talking about
![secondattack](https://user-images.githubusercontent.com/19636085/37520562-db2720e8-291d-11e8-8925-5d1ff4c8a04a.jpg)

Currently, you have to select the unit again in order to attack a second time

**Comment by Vultraz** [comment]
Obviously, the red arrows would show up again if the unit was selected. You can't have them just floating there.

**Comment by cooljeanius** [comment]
I agree that the game should do a better job of handling units with max_attacks>1, but I also think it might need a little more than just reselecting the unit... if the unit just reselected with no further context, I might think there was some sort of bug or something... maybe some sort of "attack again!" notification via floating text or whatever?
